tunelo/api/channels.py

```diff
@@ -272,51 +274,52 @@ def update_channel(uuid, patch=None):
     spoke, peers = get_channel_by_uuid(uuid)
     return create_channel_representation(spoke, peers[uuid])
 
+
 def resolve_subnet(subnet, channel_address, project_id):
     """ """
-    # First, determine if the provided subnet is a UUID or an IP address
-    subnet_is_uuid = _is_uuid(subnet)
-    if subnet_is_uuid:
-        # If the subnet is a UUID, we fetch that subnet and make sure it's valid
+
+    def _resolve_subnet_by_name_or_id(subnet_ref):
```

Comment:
I had to largely invert the flow of logic to make this function a bit more clear w/ the additional case. Now it checks for empty subnet but w/ address (to avoid picking default when the user specifies an address outside the default subnet), then CIDR, then the base case of Name/UUID.